From 001530d399af7ad6aec91ab293445530259cb7a0 Mon Sep 17 00:00:00 2001
From: Xiaowen Liu <b37945@freescale.com>
Date: Thu, 17 Jan 2013 14:20:08 +0800
Subject: [PATCH 10/50] ENGR00214328 disabe support setting video sizes.

because it will set different resolution for preview and recorder.
but driver does not support this function now.

Signed-off-by: Xiaowen Liu <b37945@freescale.com>
---
 services/camera/libcameraservice/api1/client2/Parameters.cpp | 2 --
 1 file changed, 2 deletions(-)

diff --git a/services/camera/libcameraservice/api1/client2/Parameters.cpp b/services/camera/libcameraservice/api1/client2/Parameters.cpp
index b3b3ab3..5a2488a 100644
--- a/services/camera/libcameraservice/api1/client2/Parameters.cpp
+++ b/services/camera/libcameraservice/api1/client2/Parameters.cpp
@@ -84,8 +84,6 @@ status_t Parameters::initialize(const CameraMetadata *info) {
         ALOGV("Supported preview sizes are: %s", supportedPreviewSizes.string());
         params.set(CameraParameters::KEY_SUPPORTED_PREVIEW_SIZES,
                 supportedPreviewSizes);
-        params.set(CameraParameters::KEY_SUPPORTED_VIDEO_SIZES,
-                supportedPreviewSizes);
     }
 
     camera_metadata_ro_entry_t availableFpsRanges =
-- 
1.8.0

